deploy/003_deploy_ProductApi3ReaderProxyV1.ts

```diff
@@ -75,19 +62,28 @@ module.exports = async (hre: HardhatRuntimeEnvironment) => {
   }
   log(`Proxy 2 address: ${proxy2Address}`);
 
-  let dappId2;
   if (!isLocalNetwork) {
+    let dappId1, dappId2;
```

Comment:
Why do we use the `!isLocalNetwork` above?

Reply:
There's no need to check for dappId equality locally (NETWORK=hardhat) because the script uses mock proxies for testing